misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -224,137 +194,112 @@ public static void generateAPIDocs(Project project, String output)
                                         + "%s. Cause: %s", resourcePath.toString(), resourcesDirFile.toString(),
                                 e.getMessage()), e);
                     }
                 }
                 if (BallerinaDocUtils.isDebugEnabled()) {
-                    out.println("docerina: successfully copied project resources into " + resourcesDir);
+                    out.println("docerina: successfully copied project resources into " + resourcesDirFile);
                 }
             }
         }
         // Copy docerina ui
-        File source = new File(System.getProperty("ballerina.home") + File.separator + "lib" + File.separator +
-                "tools" + File.separator + "doc-ui");
-        File dest;
-        if (source.exists()) {
-            dest = new File(output);
-            try {
-                FileUtils.copyDirectory(source, dest);
-            } catch (IOException e) {
-                out.println(String.format("docerina: failed to copy doc ui. Cause: %s", e.getMessage
-                        ()));
-                log.error("Failed to copy the doc ui.", e);
-            }
-        } else {
-            dest = new File(output, "index.html");
-            try {
-                FileUtils.copyInputStreamToFile(BallerinaDocGenerator.class
-                        .getResourceAsStream("/doc-ui/index.html"), dest);
-                //BallerinaDocUtils.unzipResources(, dest);
-            } catch (IOException e) {
-                out.println(String.format("docerina: failed to copy doc ui. Cause: %s", e.getMessage
-                        ()));
-                log.error("Failed to copy the doc ui.", e);
-            }
-        }
-
-        try {
-            String zipPath = System.getProperty(BallerinaDocConstants.OUTPUT_ZIP_PATH);
-            if (zipPath != null) {
-                if (BallerinaDocUtils.isDebugEnabled()) {
-                    out.println("docerina: generating the documentation zip file.");
+        if (!excludeUI) {
+            File source = Path.of(System.getProperty("ballerina.home"), "lib", "tools", "doc-ui").toFile();
+            File dest;
+            if (source.exists()) {
+                dest = output.toFile();
+                try {
+                    FileUtils.copyDirectory(source, dest);
+                } catch (IOException e) {
+                    log.error("Failed to copy the doc ui.", e);
                 }
-                BallerinaDocUtils.packageToZipFile(output, zipPath);
-                if (BallerinaDocUtils.isDebugEnabled()) {
-                    out.println("docerina: successfully generated the documentation zip file.");
+            } else {
+                dest = output.resolve("index.html").toFile();
+                try {
+                    FileUtils.copyInputStreamToFile(BallerinaDocGenerator.class
+                            .getResourceAsStream("/doc-ui/index.html"), dest);
```

Comment:
Will this work in windows ?

Reply:
I think it will work.
`When accessing a internal resource, like you did with getResouceAsStream, the file separator must be /.`
https://stackoverflow.com/questions/41676987/path-not-working-using-file-separator/41677152